packages/node_modules/@ciscospark/react-component-adaptive-card/src/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Adaptive Card Component renders properly 1`] = `
```

Comment:
@adamweeks  there is ref(node element) in div which I don't think can pass as props .Hence couldn't include more details . If you have any suggestions about this please let me know otherwise all other comments got resolved